solidity/random-beacon/README.adoc

```diff
@@ -289,6 +286,144 @@ operators whose nodes never participate in signing because they are
+|=== 
+^|Property Name
+^|Description
+|Governable
+|Default Value
+
+4+s|DKG
+
+|groupSize
+|Size of a group in the threshold relay
+|No
+|64
+
+|groupThreshold
+|The minimum number of group members needed to interact according to the protocol
+to provide signatures for the DKG result. 
+|No
+|33
+
+|activeThreshold
+|The minimum number of active and properly behaving group members during the DKG
+needed to accept the result.
+|No
+d|`58` +
+_90% of groupSize_
+
+|offchainDkgTime
+|Time in blocks after which DKG result is complete and ready to be published by
+clients
+|No
+d|`72 blocks` +
+_5 * (1 + 5) + 2 * (1 + 10) + 20_
+
+|resultChallengePeriodLength
+|Time in blocks during which a submitted result can be challenged
+|Yes
+d|`1440 blocks` +
+_~6h assuming 15s block time_
+
+|resultSubmissionEligibilityDelay
+|Time in blocks after which the next group member is eligible to submit DKG result
+|Yes
+|10 blocks
+
+4+s|Groups
+
+|groupLifetime
+|Group lifetime in blocks
+|Yes
+d|`80640 blocks` +
+_~2 weeks assuming 15s block time_
+
+4+s|Relay Entry
+
+|relayEntrySubmissionEligibilityDelay
+|The number of blocks it takes for a group member to become eligible to submit the
+relay entry
+|Yes
+|10 blocks
+
+|relayEntryHardTimeout
+|Hard timeout in blocks for a group to submit the relay entry
+|Yes
+d|`5760 blocks` +
+_~24h assuming 15s block time_
+
+|relayEntrySubmissionFailureSlashingAmount
+|Slashing amount for not submitting relay entry
+|Yes
+d|`1000e18` +
+_1 000 T_
+
+4+s|Random Beacon
+
+|callbackGasLimit
+|Relay entry callback gas limit
+|Yes
+|
+
+|groupCreationFrequency
+|The frequency of new group creation
+|Yes
+|
+
+|dkgResultSubmissionReward
+|Reward in T for submitting DKG result
+|Yes
+|
+
+|sortitionPoolUnlockingReward
+|Reward in T for unlocking the sortition pool if DKG timed out
+|Yes
+|
+
+|ineligibleOperatorNotifierReward
+|Reward in T for notifying the operator is ineligible
+|Yes
+|
+
+|maliciousDkgResultSlashingAmount
+|Slashing amount for supporting malicious DKG result
+|Yes
+|
+
+|unauthorizedSigningSlashingAmount
+|Slashing amount when an unauthorized signing has been proved
+|Yes
+|
+
+|sortitionPoolRewardsBanDuration
+|Duration of the sortition pool rewards ban imposed on operators who missed their
+turn for relay entry or DKG result submission
+|Yes
+|
+
+|relayEntryTimeoutNotificationRewardMultiplier
+|Percentage of the staking contract malicious behavior notification reward which
+will be transferred to the notifier reporting about relay entry timeout
+|Yes
+|
+
+|unauthorizedSigningNotificationRewardMultiplier
+|Percentage of the staking contract malicious behavior notification reward which
+will be transferred to the notifier reporting about unauthorized signing
+|Yes
+|
+
+|dkgMaliciousResultNotificationRewardMultiplier
+|Percentage of the staking contract malicious behavior notification reward which
+will be transferred to the notifier reporting about a malicious DKG result
+|Yes
+|
+
+|===
+
```

Comment:
We also have `minimumAuthorization` and `authorizationDecreaseDelay` parameters, but they don't seem to be used anywhere.
CC @pdyraga

Reply:
They will be used for T staking contract integration. This work is still in progress.

Reply:
Should they be included in this list of parameters?

Reply:
Let's add them later, once we are sure they do not need to be updated in #2747.